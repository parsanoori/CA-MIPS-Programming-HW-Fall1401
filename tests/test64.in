74
205459046
942431941
894040080
807886027
377620108
529169237
170694512
136107722
807413484
337082108
1014287601
39217461
813668359
1024828887
805290297
258264744
754548948
299086968
297797718
979860913
1051961930
304125443
245226444
527066024
432171156
836157089
634312726
509766502
1059514923
253552621
625910153
663556063
458579151
306786941
625656919
1014830588
530342426
683514863
945482291
100796526
373567312
348268450
502028665
452361930
474793981
560550852
647634096
1065865504
985842001
1044497788
94070169
940438675
219659321
388986580
522734951
6586826
918637387
28865715
257687374
437471020
96309510
281806141
683008629
874223484
57881230
1032704924
186730559
635122334
741387917
726272769
652822720
319678239
571682604
214264607