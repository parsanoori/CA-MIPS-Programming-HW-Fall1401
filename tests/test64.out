6586826
28865715
39217461
57881230
94070169
96309510
100796526
136107722
170694512
186730559
205459046
214264607
219659321
245226444
253552621
257687374
258264744
281806141
297797718
299086968
304125443
306786941
319678239
337082108
348268450
373567312
377620108
388986580
432171156
437471020
452361930
458579151
474793981
502028665
509766502
522734951
527066024
529169237
530342426
560550852
571682604
625656919
625910153
634312726
635122334
647634096
652822720
663556063
683008629
683514863
726272769
741387917
754548948
805290297
807413484
807886027
813668359
836157089
874223484
894040080
918637387
940438675
942431941
945482291
979860913
985842001
1014287601
1014830588
1024828887
1032704924
1044497788
1051961930
1059514923
1065865504